recipes/r-cubature/meta.yaml

```diff
@@ -42,7 +42,7 @@ test:
     - "\"%R%\" -e \"library('cubature')\""  # [win]
 
 about:
-  home: https
+  home: https://cran.r-project.org/web/packages/cubature/index.html
```

Comment:
For these kinds of minor changes, please use `[ci skip] [skip ci]` in the commit message title.